PresentData/PresentMonTraceConsumer.hpp

```diff
@@ -97,6 +97,9 @@ struct PresentEvent {
     Runtime Runtime;
     PresentMode PresentMode;
     PresentResult FinalState;
+    uint32_t DestWidth;

```

Comment:
These new fields are uninitialized. Add them to the initializer list in PresentMonTraceConsumer.cpp.

Reply:
Added, thanks!